.releaserc

```diff
@@ -14,9 +13,11 @@
             "path": "build/markdown/spec.md",
             "label": "Github Markdown distribution"
           }
-        ]
+        ],
+        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
```

Comment:
Programmatic commits generated on my behalf kind of give me the heebie-jeebies... __unless__ they _say_  or somehow indicate that they were generated programmatically. 

So maybe something like

```suggestion
        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}\n\nThis commit has been made automagically via .releaserc"
```

Reply:
the commits already come from @semantic-release-bot 

I'm not opposed though.

Reply:
oh, are they `authored` by the robot too?

Reply:
we do have precedent for this here:
https://github.com/open-rpc/generator-client/issues/42